examples/chat/src/main.rs

```diff
@@ -109,7 +111,7 @@ async fn websocket(stream: WebSocket, state: Arc<AppState>) {
     let mut send_task = tokio::spawn(async move {
         while let Ok(msg) = rx.recv().await {
             // In any websocket error, break loop.
-            if sender.send(Message::Text(msg)).await.is_err() {
+            if sender.send(Message::Text(msg.into())).await.is_err() {
```

Comment:
It might be worth also adding similar methods `Message::text`, `Message::binary` as upstream to make code like this a little nicer. As a mixed user of axum & tungstenite you maybe would expect them to be available in both.